components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1530,6 +1531,29 @@ def getMaxPlaneSize(self):
                 int(c.getConfigValue('omero.pixeldata.max_plane_height')))
         return self._maxPlaneSize
 
+    def getOrphanedImagesSettings(self):
+        default = {
+            "enabled": True,
+            "name": "Orphaned Images",
+            "description": ("This is a virtual container with "
+                            "orphaned images.")
+        }
```

Comment:
I missed this before, but seeing as we're using getConfigDefaults() to avoid hard coding these elsewhere, seems like that makes sense here too, or is that going to have performance implications? Since we're doing this on every login, it might make sense to leave it as is until we know this is going to be quick? cc @joshmoore.